### DISCUSSION TITLE
List Tmux windows

### DISCUSSION THREAD

**Discussion by arnold-c** [joshmedeski/sesh#139] [Q&A]
Sesh has been great, but I was wondering if there was a way to list Tmux windows, allowing for faster navigation to specific locations?

**Comment by joshmedeski** [comment]
Right now that feature doesn't exist. I personally use unnamed windows with my [nerd font window name plugin](https://github.com/joshmedeski/tmux-nerd-font-window-name) so I don't have a need for windows to show.

@omerxx has a great tool called [tmux-sessionx](https://github.com/omerxx/tmux-sessionx) that does more complex things with windows, check it out!

**Reply by arnold-c** [reply]
Thanks for the response. I will check it out!
